packages/story-editor/src/app/font/actions/useLoadFontFiles.js

```diff
@@ -42,7 +42,10 @@ function useLoadFontFiles() {
 
     switch (service) {
       case 'fonts.google.com':
-        await loadStylesheet(getGoogleFontURL([{ family, variants }], 'auto'));
+        await loadStylesheet(
+          getGoogleFontURL([{ family, variants }], 'auto'),
+          elementId
```

Comment:
Previously, without `elementId` getting passed here the id was getting set as 'undefined' which was making the stylesheets stack up with every new test set in `head` on karma runs which was contributing to early disconnect and slowing down run time.